### PR TITLE
[CBRD-23085] REGEXP_LIKE function

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-23085

**REGEXP_LIKE function**
REGEXP_LIKE (_source_str_, _regex_pattern_ [, _match_type_])

returns whether a given string match with the regex pattern

Optional arguments:
- _match_type_: A string to configure the behavior of matching
  * c: Case sensitive matching.
  * i: Case insensitive matching.

If any of the arguments is NULL, the return value is NULL.

### CODE REVIEW

**Comment by cubridci** [comment]
Build comment file: 
--------------
#### File: src/parser/type_checking.c (1 incorrect indentation(s) found)
```
 19856 
```
#### File: src/query/string_opfunc.c (1 incorrect indentation(s) found)
```
  4986 		       cub_regex_object ** comp_regex, char **comp_pattern)
```
#### File: src/query/string_opfunc.h (1 incorrect indentation(s) found)
```
   229 				  cub_regex_object ** comp_regex, char **comp_pattern);
```
--------------
**Please run 'indent -l120 -lc120' to correct the indentation**

--------------

File: src/query/string_opfunc.c [hunk 1]
```diff
@@ -4946,6 +4946,204 @@ db_string_regexp_instr (DB_VALUE * result, DB_VALUE * args[], int const num_args
...
+	    goto exit;
+	  }
+	memcpy (rx_compiled_pattern, pattern_string.c_str (), pattern_length);
+	rx_compiled_pattern[pattern_length] = '\0';
+
+	error_status = cubregex::compile (rx_compiled_regex, rx_compiled_pattern, reg_flags, collation);
+	if (error_status != NO_ERROR)
+	  {
+  error_status = (error_status == ER_QSTR_BAD_SRC_CODESET) ? NO_ERROR : error_status;
+  goto exit;
```
Comment (shparkcubrid) [inline src/query/string_opfunc.c]: Indent?

**Comment by shparkcubrid** [comment]
Congratulations on completing your regexp-related functions :+1:

File: src/query/string_opfunc.c (hunk 1 above)
Comment (hgryoo) [reply src/query/string_opfunc.c]: I fixed indent

**Pull request merged**